kafka/adminapi.go

```diff
@@ -1555,15 +1641,21 @@ func (a *AdminClient) SetSaslCredentials(username, password string) error {
 
 // Close an AdminClient instance.
 func (a *AdminClient) Close() {
+	if a.isClosed {
+		return
+	}
+
```

Comment:
To avoid a race condition this should be a cmp and exchange, or perhaps an increment (if they are atomic in go).
Two different threads calling Close() at the same time now risks the below code to execute in parallel and thus crash.

Reply:
got it. will use atomic.CompareAndSwapUint32()